plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/impl/api/statistics.kt

```diff
@@ -86,39 +85,88 @@ private fun Arguments.createUpdatedColumn(
 }
 
 val skipNaN = true
+val ddofDefault: Int = 1
+val percentileArg: Double = 30.0
 val sum = Aggregators.sum(skipNaN)
+val mean = Aggregators.mean(skipNaN)
+val std = Aggregators.std(skipNaN, ddofDefault)
+val median = Aggregators.median(skipNaN)
+val min = Aggregators.min<Double>(skipNaN)
+val max = Aggregators.max<Double>(skipNaN)
+val percentile = Aggregators.percentile(percentileArg, skipNaN)
 
 /** Adds to the schema only numerical columns. */
-abstract class Aggregator0 : AbstractSchemaModificationInterpreter() {
+abstract class Aggregator0(val aggregator: Aggregator<*, *>) : AbstractSchemaModificationInterpreter() {
     private val Arguments.receiver: PluginDataFrameSchema by dataFrame()
 
     override fun Arguments.interpret(): PluginDataFrameSchema {
         val resolvedColumns = receiver.columns()
             .filterIsInstance<SimpleDataColumn>()
             .filter { it.type.type.isSubtypeOf(session.builtinTypes.numberType.type, session) }
 
-        val newColumns = generateStatisticResultColumns(sum, resolvedColumns)
+        val newColumns = generateStatisticResultColumns(aggregator, resolvedColumns)
 
         return PluginDataFrameSchema(receiver.columns() + newColumns)
     }
 }
 
-/** Implementation for `sum`. */
-class Sum0 : Aggregator0()
+class Sum0 : Aggregator0(sum)
+
+class Mean0 : Aggregator0(mean)
+
+class Std0 : Aggregator0(std)
 
 /** Adds to the schema only numerical columns. */
-abstract class Aggregator1 : AbstractSchemaModificationInterpreter() {
+abstract class AggregatorIntraComparable0(val aggregator: Aggregator<*, *>) : AbstractSchemaModificationInterpreter() {
+    private val Arguments.receiver: PluginDataFrameSchema by dataFrame()
+
+    override fun Arguments.interpret(): PluginDataFrameSchema {
+        val resolvedColumns = receiver.columns()
+            .filterIsInstance<SimpleDataColumn>()
+            .filter { isIntraComparable(it, session) }
+
+        val newColumns = generateStatisticResultColumns(aggregator, resolvedColumns)
+
+        return PluginDataFrameSchema(receiver.columns() + newColumns)
```

Comment:
I think it should be `PluginDataFrameSchema(newColumns)`
![image](https://github.com/user-attachments/assets/79e6992f-8a2b-4ddb-9ccf-2f5e37627b9e)

![image](https://github.com/user-attachments/assets/901636a8-46d1-4754-be1b-43bd4cc34c9f)

df.compareSchemas(strict = true) can help detect if compile schema has columns that don't exist in runtime for such cases